src/main/resources/application.conf

```diff
@@ -1,25 +1,3 @@
-
-methods {
-  baseUrl="https://agora.dsde-dev.broadinstitute.org"
-}
-
-workspace {
-  baseUrl="https://rawls.dsde-dev.broadinstitute.org"
-}
-
-openam {
-  deploymentUri = "https://openam-ci.vault.broadinstitute.org/opensso"
-  realm = "/apollo"
-  // The next three values should be provided by your environment (e.g. in an .sbtopts file)
-  username = ""
-  password = ""
-  commonName = ""
-  authIndex {
-    type = "module"
-    value = "DataStore"
-  }
-}
-
```

Comment:
at this point there's really nothing left in application.conf. The akka section is just boilerplate defaults in case we ever want to change them. We could remove the file altogether, or move all the info from configurations.conf into here.